dataproc/instantiate_inline_workflow_template.go

```diff
@@ -0,0 +1,115 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package dataproc shows how you can use the Cloud Dataproc Client library to manage
+// Cloud Dataproc clusters. In this example, we'll show how to instantiate an inline
+// workflow template.
```

Comment:
Only need the package comment in one file. So, you should be able to delete this.

Reply:
If one is looking to learn more about the package, how does one differentiate if they should check this sample or the other one? Should these all be separate packages?

Reply:
https://pkg.go.dev/github.com/GoogleCloudPlatform/golang-samples/dataproc?tab=doc

If there are many files (like for samples), it's common to have a `doc.go` file that has this comment, the `package` line, and nothing else.